src/Magick.NET/Helpers/ByteArrayWrapper.cs

```diff
@@ -3,34 +3,76 @@
 
 using System;
 using System.IO;
+#if NETSTANDARD2_1_OR_GREATER
```

Comment:
This check should be `#if !NETSTANDARD2_0` instead to be in line with the rest of the project.